### PR TITLE
BaSP-TG-40: Employee

### PR DESCRIPTION
**Pull request by EzeRocco**
- [x]  Implement Rows
- [x]  Implement Fild
- [x] Implement Button
- [x]  Implement Modal
- [x] Implement Realoader

### CODE REVIEW

**Review by martumoser — CHANGES_REQUESTED** [review]
Implementar los cambios incorporados en los formularios y en el edit y volver a solicitar review

File: src/Components/Employees/index.js [hunk 1]
```diff
@@ -1,86 +1,115 @@
...
   };
 
   // PUT EMPLOYEE
 
-  const editEmployee = (body) => {
-    const employeesUpdated = employees.map((employee) => {
-      if (employee._id === body._id) {
-        return body;
+  const editItem = (data) => {
+    const EmpnUpd = list.map((employee) => {
```
Comment (martumoser) [inline src/Components/Employees/index.js:62]: usa camelCase aca

**Review by martumoser — CHANGES_REQUESTED** [review]

**Review by LucasPrat1 — APPROVED** [review]

**Review by LucianoAlarcon — APPROVED** [review]

**Review by CristianAlcober — CHANGES_REQUESTED** [review]
Hi Eze! Almost everything works as intended, however there are a few issues:

- [x] Whether you click to add an employee or edit one, the modal does not appear at the center of the screen, instead it appears lower and it prevents the user from using the buttons to 'add' or 'cancel'.
- [x] The 'Name' fieldset stays blank and does not let you fill it with anything.
- [ ] When you send an employee with a wrong or empty fieldset (I did it using tab to reach the button, since I can't see the buttons) the modal appears empty with no message (however, this modal does show at the center of the screen).

Those are all the issues I could find, please fix them as soon as you can. Everything else works wonderfully!

**Review by CristianAlcober — APPROVED** [review]
Excellent job Eze! Everything works nicely!

**Review by martumoser — APPROVED** [review]

**Pull request merged**